TopShelf.Owin/Properties/AssemblyInfo.cs

```diff
@@ -3,13 +3,13 @@
 [assembly: AssemblyTitle("TopShelf.Owin")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyProduct("TopShelf.Owin")]
-[assembly: AssemblyCopyright("Copyright ©  2014")]

```

Comment:
Actually, I think we should remove `AssemblyCopyright` as it is licensed under MIT